src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
 public class Application {
+    private static int count = 3;
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        while (count > 0) {
+            Lottery winCar = new Lottery();
```

Comment:
[Don’t create several instances of the same class if you can use only one instance for your purposes. ](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#dont-create-several-instances-of-the-same-class-if-you-can-use-only-one-instance-for-your-purposes)